packages/sirius-web/frontend/sirius-web-application/src/extension/DefaultExtensionRegistry.tsx

```diff
@@ -629,4 +632,25 @@ defaultExtensionRegistry.putData<DiagramNodeActionOverrideContribution[]>(
   }
 );
 
+/*******************************************************************************
+ *
+ * Palette ellipse node appearance contribution
+ *
+ * Used to contribute custom node eappearance section on the palette for ellipse nodes
```

Comment:
eappearance => appearance
(do not hesitate to check all your code with a spell checker in VSCode (e.g. Code Spell Checker) or any other IDE, it will show you all typos)